vocabularies/Common.xml

```diff
@@ -622,6 +622,9 @@ The minimum structure is
         <Property Name="CollectionRoot" Type="Edm.String" Nullable="true">
           <Annotation Term="Core.Description" String="Service root of the value list collection; not specified means local to the document containing the annotation" />
         </Property>
+        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" Nullable="true" DefaultValue="true"> 
```

Comment:
```suggestion
        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" Nullable="true"> 
```
A `DefaultValue` for a term property makes no sense, because it is considered only in POST und PUT payloads: [https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_DefaultValue](OData-CSDL, sec. 7.2.7)

Reply:
It makes sense, the cited specification passage says
> A primitive or enumeration property MAY define a default value that is used if the property is not explicitly represented in an annotation

However: the default value should be `false` because we cannot assume all existing value helps to suddenly support distinct values.

Reply:
Should have read more closely...